js/src/error.ts

```diff
@@ -4,6 +4,9 @@
 export type Error = {
     code: ErrorCode;
     message: string; // Human-readable representation of the error
+    readerReadyDuration?: number; // Elapsed time since launchAsync call until reader request succeeded
```

Comment:
It was not expected that these changes would go into Public SDK. They were Internal SDK only

Reply:
Main thing we needed in the Public SDK update were the microphone permissions update